subprojects/docs/src/samples/userguide/tasks/incrementalTask/groovy/build.gradle

```diff
@@ -62,6 +62,8 @@ class IncrementalReverseTask extends DefaultTask {
 
         // tag::out-of-date-inputs[]
         inputs.outOfDate { change ->
+            if (change.file.directory) return
```

Comment:
Does that mean that no directories have been returned before?

Reply:
Yeah, though the old behavior was a bug since incremental inputs included directory changes. TBH, I'd like remove all directories from showing up in the inputs.